### PR TITLE
Update plausible.yaml to add ports

### PR DESCRIPTION
**Pull request by commandodev**
Needed to override the defaults `127.0.0.1:8000:8000` to simply `8000:8000`

This was needed for me to get plausible's websockets working

### CODE REVIEW

**Comment by commandodev** [comment]
I also had to regenerate the `SERVICE_BASE64_TOTP` env var with `openssl rand -base64 32` as the one that got added to the service wasn't the right length.

File: templates/compose/plausible.yaml [hunk 1]
```diff
@@ -17,6 +17,8 @@ services:
       - plausible_db
       - plausible_events_db
       - mail
+    ports:
+     - 8000:000
```
Comment (JonasPetry) [inline templates/compose/plausible.yaml:21]:
```suggestion
     - 8000:8000
```

**Comment by andrasbacsai** [comment]
Why is this needed? Why would you like to map the port 8000 to the host system?

**Comment by commandodev** [comment]
I broke my install and had to re-create it and this time the original docker compose worked as is. So I'm closing this PR

**Pull request closed without merging**